### PR TITLE
User story 41/votingscreen

### LINKED ISSUE #41: As a player, I can see all prompts and drawings from my game so that i know what was drawn

### PR DESCRIPTION
**Pull request by ChristopherPearse**
Voting screen implementation. Can view all players prompts and pictures drawn during the round. Cannot vote yet. 

Closes  #41

### CODE REVIEW

File: src/controller/userController.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ exports.createUserAccount = async (req, res) => {
 
   try {
     const result = await db.query(query, values)
-    return res.redirect('/draw')
+    return res.redirect('/landing')
```
Comment (RudolphTorlage) [inline src/controller/userController.js:11]: send un-logged in users to landing page

File: src/server.js [hunk 2]
```diff
@@ -103,7 +112,17 @@ io.on('connection', (socket) => {
...
+      roomId,
+      users.get(socket.id).username,
+      'drawing',
+      image,
+      socket.id
+    )
+    console.log('UserID: ', users.get(socket.id).id)
+    console.log('SID: ', socket.id)
+    console.log('SIDGet', users.get(socket.id))
+    console.log('uSERmAp', users)
```
Comment (RudolphTorlage) [inline src/server.js:125]: what is uSERmAp?

**Review by RudolphTorlage — CHANGES_REQUESTED** [review]
Variable naming is inconsistent, and logs are extensive. Please fix this in the server.js file line 125

File: src/server.js (hunk 2 above)
Comment (ChristopherPearse) [reply src/server.js]: Apologies, it was meant to be for me to bug test, however, I erroneously forgot to remove it. All console logs for such purposes have been removed.

Comment (RudolphTorlage) [reply src/server.js]: thank you kindly

**Review by RudolphTorlage — APPROVED** [review]
Adds functionality to review game and build on for future voting procedure.

**Pull request merged**